### ISSUE TITLE
TinyMCE: fix dark theme in Insert Template

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#7225]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] The issue is in the develop branch
- [ ] The issue is in the latest released 4.1.x
- [ ] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

The dark theme for the "Insert Template" option in TinyMCE is incorrect, this only happens when editing content in XB.

### Steps to reproduce

Steps:

1. Create a site from Editorial
2. Update the User Interface config to have the Insert template. 

```
<tinymceOptions>
                        <![CDATA[
                            {
                                "menubar": true,
                                "theme": "silver",
                                "plugins": "preview searchreplace autolink directionality visualblocks visualchars fullscreen image link media template codesample table charmap pagebreak nonbreaking anchor insertdatetime advlist lists wordcount help acecode",
                                "extended_valid_elements": "",
                                "valid_children": "",
                                "toolbar1": "template | formatselect | bold italic strikethrough forecolor backcolor | link | alignleft aligncenter alignright alignjustify | numlist bullist outdent indent | removeformat",
                                "code_editor_wrap": false,
                                "toolbar_sticky": true,
                                "image_advtab": true,
                                "encoding": "xml",
                                "relative_urls": false,
                                "remove_script_host": false,
                                "convert_urls": false,
                                "remove_trailing_brs": false,
                                "media_live_embeds": true,
                                "autoresize_on_init": false,
                                "autoresize_bottom_margin": 0,
                                "menu": {
                                    "tools": { "title": "Tools", "items": "tinymcespellchecker code acecode wordcount" }
                                },
                                "automatic_uploads": true,
                                "file_picker_types":  "image media file",
                                "paste_data_images": true,
                                "templates": [{ "title": "Template 1", "description": "Template 1 description", "content": "Template 1 content" },
                                { "title": "Template 12", "description": "Template 12 description", "content": "Template 12 content" }],                                "content_css": [],
                                "content_style": "body {}",
                                "contextmenu": false
                            }
                        ]]>
                    </tinymceOptions>
```

3.Edit an RTE section in XB
4. See the issue

### Screenshots and/or videos

4.1.6 
![XB Insert Template 4 1 6](https://github.com/user-attachments/assets/262d88fe-a470-4197-a8c7-20ce36305d30)

4.2.0
![image](https://github.com/user-attachments/assets/fb2c08da-a413-4f5e-b83a-f67603ab18db)

**Comment by rart** [comment]
@jvega190 let's see if we can theme that more appropriately. 4.1.6 didn't look good either, it should be better than that.

**Comment by yacdaniel** [comment]
Verified fixed in `4.2.0`

![image](https://github.com/user-attachments/assets/b60ecafb-5fee-4a5a-a0ad-04d45427fecc)
